runtime/src/iree/builtins/ukernel/tools/pack_test.cc

```diff
@@ -24,12 +24,12 @@ static void iree_pack_reference(const iree_uk_pack_params_t& params) {
   iree_uk_ssize_t tile_size0 = params.out_size2;
   iree_uk_ssize_t tile_size1 = params.out_size3;
   iree_uk_ssize_t out_stride_l0 = params.out_stride0;
-  iree_uk_ssize_t out_stride_l1 = params.out_size3 * params.out_size2;
+  iree_uk_ssize_t out_stride1 = params.out_size3 * params.out_size2;
```

Comment:
was this intentional? (out_stride_l0, out_stride1, out_stride_l2, out_stride_l3)